rosdep/base.yaml

```diff
@@ -4545,6 +4545,26 @@ muparser:
   fedora: [muParser-devel]
   gentoo: [dev-cpp/muParser]
   ubuntu: [libmuparser-dev]
+nanoflann:
+  arch: [nanoflann]
+  debian: [libnanoflann-dev]
+  gentoo: [sci-libs/nanoflann]
+  ubuntu:
+    bionic: null
+    artful: null
```

Comment:
We don't need to fill in all the now unsupported targets. So the only ones we need to exclude bionic and focal. All the others are EOL so we don't need to add complexity to support them.